rosdep/base.yaml

```
@@ -2083,6 +2083,11 @@ libgps:
   gentoo: [sci-geosciences/gpsd]
   opensuse: [gpsd-devel]
   ubuntu: [libgps-dev]
+libgrpc-dev:
+  arch: [grpc]
+  debian: [libgrpc-dev]
```

Filling in Chris' suggestion more concretely.
```suggestion
  debian:
    '*': [libgrpc-dev]
    stretch: null
```